ACE/ace/OS_NS_Thread.cpp

```diff
@@ -3615,6 +3615,7 @@ ACE_OS::thr_create (ACE_THR_FUNC func,
 
 # if defined (ACE_HAS_PTHREADS)
   int result;
+  int name_set = 0;
```

Comment:
Can be bool and only needs to be there when ACE_HAS_PTHREAD_SETNAME_NP is defined